client/index.html

```diff
@@ -51,8 +51,13 @@
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
 
   <link rel=stylesheet href="css/style.css" />
+<<<<<<< Updated upstream
```

Comment:
You commited a stash conflict piece here

Reply:
oops